apps/wing/src/commands/test/test.ts

```diff
@@ -195,12 +195,12 @@ async function testOne(
 
   let results: std.TestResult[] = [];
   if (shouldExecute) {
+    options.rootId = options.rootId ?? `Test.${nanoid(10)}`;
```

Comment:
Not sure I understand why this change is needed?

It's better to put the id in a local variable and reuse than to mutate the options object